apps/src/applab/applab.js

```diff
@@ -103,6 +106,18 @@ function loadLevel() {
   Applab.softButtons_ = level.softButtons || {};
   Applab.appWidth = level.appWidth || defaultAppWidth;
   Applab.appHeight = level.appHeight || defaultAppHeight;
+  // In share mode we need to reserve some number of pixels for our in-app
+  // footer. We do that by making the play space slightly smaller elsewhere.
+  // Applab.appHeight represents the height of the entire app (footer + other)
+  // Applab.footerlessAppHeight represents the height of only the "other"
+  if (Applab.appHeight > 480) {
+    throw new Error('Strange things may happen with appHeight > 480');
+  }
+  if (Applab.appHeight === 480) {
+    Applab.footerlessAppHeight = Applab.appHeight - FOOTER_HEIGHT;
+  } else {
+    Applab.footerlessAppHeight = Applab.appHeight;
```

Comment:
I am concerned that hard-coding 480 here will limit an otherwise flexible thing. AFAIK, today we can create a new applab level with any dimensions and it will work. after this change I'm not sure what the behavior will be.